model/riscv_sys_control.sail

```diff
@@ -40,6 +40,15 @@ function check_Counteren(csr : csreg, p : Privilege) -> bool = {
   feature_enabled_for_priv(p, mcounteren.bits[index], scounteren.bits[index])
 }
 
+// Return true if the stimecmp[h] CSR is accessible OR the CSR is not stimecmp[h].
+function check_Stimecmp(csr : csreg, p : Privilege) -> bool = {
+  // Check if it is not stimecmp.
+  if csr != 0x14D & csr != 0x15D
+  then return true;
```

Comment:
I think this would look better with the then on the preceding line like:
```
  if csr != 0x14D & csr != 0x15D then {
    return true
  };
```
or maybe with the whole if as a single line.

Reply:
I never know how to format these - where's that autoformatter? :-D

I changed it to a single line.